File: solutions/week10/aufgabe_10_06.py
'''
Aufgabe 10.6

Was könnte folgender Code für einen Zweck haben?

siege_computer = 0
if wahl_computer == 'Stein':
    if wahl_benutzer == 'Stein':
        print('Unentschieden, ich hatte ebenfalls Stein.')
    elif wahl_benutzer == 'Schere':
        print('Punkt für mich, Stein schleift Schere.')
        siege_computer += 1
    else # wahl_benutzer == 'Papier'
        print('Punkt für Sie, Papier umwickelt Stein.')
        siege_benutzer +=1
Korrigieren sie die 4 Fehler.
'''

siege_computer = 0
siege_benutzer = 0
wahl_computer = 'Stein'
wahl_benutzer = 'Stein'
if wahl_computer == 'Stein':
    if wahl_benutzer == 'Stein':
        print('Unentschieden, ich hatte ebenfalls Stein.')
    elif wahl_benutzer == 'Schere':
        print('Punkt für mich, Stein schleift Schere.')
        siege_computer += 1
    else: # wahl_benutzer == 'Papier'
        print('Punkt für Sie, Papier umwickelt Stein.')
        siege_benutzer +=1

File: solutions/week10/aufgabe_10_06.m
siege_computer = 0;
wahl_computer = 'Stein';
wahl_benutzer = 'Papier';
siege_benutzer = 0;
if strcmp(wahl_computer, 'Stein')
    if strcmp(wahl_benutzer, 'Stein')
        disp('Unentschieden, ich hatte ebenfalls Stein.');
    elseif strcmp(wahl_benutzer, 'Schere')
        disp('Punkt für mich, Stein schleift Schere.');
        siege_computer = siege_computer + 1;
    else % wahl_benutzer == 'Papier'
        disp('Punkt für Sie, Papier umwickelt Stein.');
        siege_benutzer = siege_benutzer + 1;
    end
end